pkg/networkservice/mechanisms/vxlan/common.go

```diff
@@ -84,7 +84,7 @@ func addDel(ctx context.Context, conn *networkservice.Connection, vppConn api.Co
 			WithField("vppapi", "AddNodeNext").Debug("completed")
 
 		now = time.Now()
-		vxlanAddDelTunnel := &vxlan.VxlanAddDelTunnelV2{
+		vxlanAddDelTunnelV3 := &vxlan.VxlanAddDelTunnelV3{
```

Comment:
I believe V3 behaves the same as V2 as long as parameter `is_l3` is false (which is the default):
https://github.com/FDio/vpp/commit/3e38422ab905d26ab1625c74268e30c94327ea54

I'm not sure I see the point of renaming vxlanAddDelTunnel though.

Reply:
I agree, I just overreacted, and for first view looked good to rename the variable name also.
For the parameters, exactly only isL3 is the extra in V3, and by default that is false:
https://github.com/FDio/govpp/blob/master/binapi/vxlan/vxlan.ba.go#L380